example/fittemplate.md

```diff
@@ -0,0 +1,250 @@
+    polhemus_sphere = ft_prepare_headmodel(cfg, polhemus.fid);
+
+Determine the global scaling and translation
+
+    scale = polhemus_sphere.r/template_sphere.r;
+
+    T2 = [1 0 0 template_sphere.o(1);
+          0 1 0 template_sphere.o(2);
+          0 0 1 template_sphere.o(3);
+          0 0 0 1                ];
+
+    T1 = [1 0 0 -template_sphere.o(1);
+          0 1 0 -template_sphere.o(2);
+          0 0 1 -template_sphere.o(3);
+          0 0 0 1                 ];
+
+    S  = [scale 0 0 0;
+          0 scale 0 0;
+          0 0 scale 0;
+          0 0 0 1 ];
+
+    transformation = T1*S*T2;
+
+Apply the transformation to the template head model
+
+    template_fiducials = ft_transform_geometry(transformation, template);
+
+### Method 3: On the basis of the full head surface
+
+This requires an external toolbox, which can be downloaded [here](https://sites.google.com/site/myronenko/research/cpd)
+
+With this method we apply an affine transformation to the template head model to fit the
+whole Polhemus head shape.  This not only applies a translation and rotation, but also a
+scaling in the different directions and some skewing.
+
+To create the individualized head model with this method it is important that the scalp
+surface of the template head model only contains features that are also in the Polhemus head
+shape, and vice versa. We can use **[ft_defacemesh](/reference/ft_defacemesh)** to remove some features.
+
+So first we visualise both meshes:
+
+    ft_plot_mesh(template.bnd);  
+    ft_plot_mesh(polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/template_surface_polhemus_surface_plot.png" %}
+
+As the template model and the Polhemus measurement have exclusive features. The Polhemus measurement digitized facial details and part of the upper headsurface, it did not measure the head surface on the side and lower part of the back of the head. So we will now remove these from the template.
+
+    defaced_template      = template.bnd(1);
+    defaced_template.unit = template.unit
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_template =  ft_defacemesh(cfg, defaced_template);
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_polhemus =  ft_defacemesh(cfg, polhemus);
+
+Now we have another look at the surfaces
+
+    ft_plot_mesh(defaced_template);  
+    ft_plot_mesh(defaced_polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/defaced_template_surface_polhemus_surface_plot.png" %}
+
+We will now use the template scalp and the Polhemus head shape to determine the
+affine transformation.
+
+    cfg              = [];
+    cfg.headshape    = defaced_polhemus;
+    cfg.template     = defaced_template;
+    cfg.method       = 'fittemplate';
+    template_surface = ft_prepare_mesh(cfg, template.bnd);
```

Comment:
does the template now contain all three meshes, or only the scalp surface? Elsewhere you apply the transformation to the full template, i.e. three meshes.